### PR TITLE
docs: Update note format to use pymdownx.details, add footer

### PR DESCRIPTION
**Pull request by ejlee3**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

- Update documentation to use `!!! Note` format - also split out Example, Success, Failure where I found them
- Added a footer to help with navigation

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable

Run `make serve-docs`

Ensure that all the notes are formatted properly.

### CODE REVIEW

File: docs_src/pipelinerun.md [hunk 1]
```diff
@@ -207,21 +214,15 @@ The **results/pipeline0.log** file lists FPS during pipeline run.
...
-**Failure**
-    
-Review the console output for errors if you do not see all the Docker* containers. Sometimes dependencies fail to resolve. Address obvious issues and [rerun the pipeline](./pipelinerun.md#start-pipeline).
+!!! Failure
+    Review the console output for errors if you do not see all the Docker* containers. Sometimes dependencies fail to resolve. Address obvious issues and [rerun the pipeline](./pipelinerun.md#start-pipeline).
     
 ---
     
-### Stop Pipeline
+## Stop Pipelines
```
Comment (NeethuES-intel) [inline docs_src/pipelinerun.md:222]: should this remain as pipeline, as everywhere on the top we are referring to single pipeline ? or if we change it here then change ones on top to pipelines - eg `Start Pipelines` ?

Comment (ejlee3) [reply docs_src/pipelinerun.md:222]: let me change it back.

**Review by antoniomtz — APPROVED** [review]
LGTM

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by vli11 — APPROVED** [review]

**Review by brian-intel — APPROVED** [review]

**Pull request merged**